### PR TITLE
chore: fix build on Windows

### PR DESCRIPTION
**Pull request by matyasf**
not fully fixed yet, cant commit due to https://github.com/conventional-changelog/commitlint/issues/4106

what works:
- `bootstrap`
- `dev`
- `test:vitest` almost  (1 test fails for some reason)

### CODE REVIEW

File: packages/__docs__/package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   "homepage": "https://instructure.design",
   "bugs": "https://github.com/instructure/instructure-ui/issues",
   "scripts": {
-    "prestart": "npm run build:scripts:ts && node -e 'import(\"./lib/build-docs.mjs\").then(file => file.buildDocs());'\n",
```
Comment (matyasf) [inline packages/__docs__/package.json:14]: Windows does not like such a fancy command at all :/ had to move it to a file

File: packages/ui-scripts/lib/icons/build-icons.js [hunk 2]
```diff
@@ -42,9 +43,9 @@ export default {
   },
   handler: async (argv) => {
     const configFile = path.join(process.cwd(), argv.config)
-    const config = await import(configFile)
+    const configFileURL = pathToFileURL(configFile)
+    const config = await import(configFileURL)
```
Comment (matyasf) [inline packages/ui-scripts/lib/icons/build-icons.js:47]: `import()` for files must begin with `file://`, this is added by this built in helper

File: packages/__docs__/build-docs.mjs [hunk 3]
```diff
@@ -0,0 +1,4 @@
+// scripts must be built from .ts files before this is run
+import {buildDocs} from './lib/build-docs.mjs'
+
+buildDocs()
```
Comment (balzss) [inline packages/__docs__/build-docs.mjs:4]: was this file created in windows? I think this file ending is not compatible with unix EOF. I think editing this file in macos should solve this

Comment (balzss) [reply packages/__docs__/build-docs.mjs:4]: also the same file name with `lib/build-docs.mjs` is a bit confusion. I'm wondering if there's that file already, can't that one be run instead?

**Review by balzss — CHANGES_REQUESTED** [review]

File: packages/__docs__/build-docs.mjs (hunk 3 above)
Comment (matyasf) [reply packages/__docs__/build-docs.mjs:4]: There is a way not to create a new file and not use fancy syntax in `package.json` but its also ugly: https://stackoverflow.com/a/75760840 What do you think?

Comment (matyasf) [reply packages/__docs__/build-docs.mjs]: fixed it, now it doesn't need an extra file

**Review by HerrTopi — APPROVED** [review]

**Review by balzss — APPROVED** [review]

**Pull request merged**